src/test/scala/com/cognite/sdk/scala/v1/FilesTest.scala

```diff
@@ -1,5 +1,6 @@
 package com.cognite.sdk.scala.v1
 
+import java.io
```

Comment:
Better to `import java.io.{ByteArrayOutputStream, FileInputStream}` (this is what IDEA does if you use it's import magic).